File: PracticeAndroid/AndroidPractice2/Task/DynamicDemo/OnDemand/src/main/java/com/example/ondemand/MainActivity.java
package com.example.ondemand;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.os.Bundle;

import com.google.android.play.core.splitcompat.SplitCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    @Override
    protected void attachBaseContext(Context newBase) {
        super.attachBaseContext(newBase);
        SplitCompat.install(this);
    }
}

File: PracticeAndroid/Andorid_Kotlin/ADExpress/NewBarcodeScanApplication/app/src/main/java/com/androdevdk/newbarcodescanapplication/MainActivity.kt
package com.androdevdk.newbarcodescanapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.google.mlkit.vision.barcode.BarcodeScanner
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}